### DISCUSSION TITLE
Set the detail for ImageContent

### DISCUSSION THREAD

**Discussion by jmurkothintel** [microsoft/semantic-kernel#9062] [Q&A] [answered]
We are using Semantic Kernel 18.2 and GPT-4O models . Though the underlying api supports setting the detail for the image, doesn't look like Semantic Kernel allows us to do so . Would like to know from the team if there is a way around this or if it is something that is coming in future releases

**Comment by rogerbarreto** [comment, marked as answer]
We are tracking this and is coming out in a future release, we have a tracking on this task for `Ignite` which as our target should be ready before Microsoft Ignite event this November.

 - https://github.com/microsoft/semantic-kernel/issues/4759
